parser/src/grammar.lalrpop

```diff
@@ -400,6 +394,12 @@ Expr: Expr<Id> = {
 
         Expr::TypeBindings(bindings, Box::new(body))
     },
+
+    "OpenBlock" <exprs: (<SpExpr> "Semi")*> <last: SpExpr> "CloseBlock" => {
+        let mut exprs = exprs;
+        exprs.push(last);
+        Expr::Block(exprs)
+    },
```

Comment:
I thought that seeing as this is atomic (does not require parens) it makes more sense lower down. How was it breaking it?